app/connectors/ArrivalMovementConnector.scala

```diff
@@ -30,25 +29,20 @@ import scala.xml.{Node, XML}
 
 class ArrivalMovementConnector @Inject() (
   config: FrontendAppConfig,
-  http: HttpClientV2,
-  phaseConfig: PhaseConfig
+  http: HttpClientV2
 ) {
 
-  private val version = phaseConfig.values.apiVersion
-
   def getMessageMetaData(arrivalId: ArrivalId)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Messages] = {
     val url = url"${config.commonTransitConventionTradersUrl}movements/arrivals/${arrivalId.value}/messages"
     http
       .get(url)
-      .setHeader(ACCEPT -> s"application/vnd.hmrc.$version+json")
       .execute[Messages]
   }
 
   def getMessage(arrivalId: ArrivalId, messageId: String)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[Node] = {
     val url = url"${config.commonTransitConventionTradersUrl}movements/arrivals/${arrivalId.value}/messages/$messageId/body"
     http
       .get(url)
-      .setHeader(ACCEPT -> s"application/vnd.hmrc.$version+xml")
```

Comment:
We still need this header, but now it will just always be
`.setHeader(ACCEPT -> s"application/vnd.hmrc.2.1+xml")`